### PR TITLE
Merging Release/8.0 into develop after clang reformatting

### PR DESCRIPTION
**Pull request by guilpier-code**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -283,6 +295,21 @@ cmake --build _build --config release -j8
...
+cmake -B _build -S [antares_src] -DDEPS_INSTALL_DIR=<deps_install_dir> -DCMAKE_BUILD_TYPE=release -DUSE_PRECOMPILED_EXT=ON..
+```
+- Build
+ ```
+cmake --build _build --config release -j8
+```
+Note :
+> Compilation can be done on several processors with ```-j``` option.
+
+## [Unit tests](#unit-tests)
```
Comment (jmkerloch) [inline INSTALL.md:312]: Remove Unit tests paragraph

File: src/libs/antares/jit/jit.hxx [hunk 2]
```diff
@@ -68,11 +68,12 @@ void JIT::just_in_time_manager::unload_matrix_properly_from_memory(const Matrix<
 
     // - jit activated :
     //		If JIT (Just-In-Time) is activated, we have to unload data to keep the memory for
-    //the solver. 		The old width must only be kept when the matrix has fixed dimension.
+    // the solver. 		The old width must only be kept when the matrix has fixed dimension.
```
Comment (jmkerloch) [inline src/libs/antares/jit/jit.hxx:71]: clang was probably not passed on this file

File: src/libs/antares/study/parts/hydro/container.cpp [hunk 3]
```diff
@@ -174,9 +174,9 @@ bool PartHydro::LoadFromFolder(Study& study, const AnyString& folder)
 
             //	GUI part patch :
             //		We need to know, when estimating the RAM required by the solver, if the
-            //current
-            // area 		is hydro modulable. Therefore, reading the area's daily max power at this
-            // stage is 		necessary.
+            // current
```
Comment (jmkerloch) [inline src/libs/antares/study/parts/hydro/container.cpp:177]: why is there change after clang ?

File: src/libs/antares/study/study.cpp [hunk 4]
```diff
@@ -555,7 +555,7 @@ void Study::getNumberOfCores(const bool forceParallel, const uint nbYearsParalle
 
     // GUI : storing minimum number of parallel years (in a set of parallel years).
     //		 Useful in the run window's simulation cores field in case parallel mode is enabled
-    //by user.
+    // by user.
```
Comment (jmkerloch) [inline src/libs/antares/study/study.cpp:558]: same comment

File: src/solver/simulation/solver.hxx [hunk 5]
```diff
@@ -1,1636 +1,1644 @@
...
-        pAnnualCostsStatistics.writeToOutput();
-    }
-}
-
-} // namespace Simulation
-} // namespace Solver
-} // namespace Antares
-
-#endif // __SOLVER_SIMULATION_SOLVER_HXX__
+/*
```
Comment (jmkerloch) [inline src/solver/simulation/solver.hxx:1]: There was no change in this file. See if clang was passed.

File: src/solver/ts-generator/xcast/xcast.h [hunk 6]
```diff
@@ -153,18 +153,18 @@ class XCast final : private Yuni::NonCopyable<XCast>
...
     **
     ** \code
-    ** code loi		a				b			g				d       autres contraintes
-    *Loi marginale
-    ** 1		quelconque     quelconque  g> INFININ/2   d<INFINIP/2			g<=d		 Uniforme
-    *sur l'intervalle	(g,d)
-    ** 2			a>0				b>0    g> INFININ/2   d<INFINIP/2		    g<=d		 Bêta
-    *(a,b)sur l'intervalle (g,d)
+    ** code loi		a				b			g				d       autres
+    *contraintes Loi marginale
```
Comment (jmkerloch) [inline src/solver/ts-generator/xcast/xcast.h:157]: same comment

File: src/solver/utils/ortools_utils.cpp [hunk 7]
```diff
@@ -1,538 +1,544 @@
...
+
+    _solverLinearProblemOptimMap[OrtoolsSolver::glop_cbc]       =
+    MPSolver::OptimizationProblemType::GLOP_LINEAR_PROGRAMMING;
+    _solverMixedIntegerProblemOptimMap[OrtoolsSolver::glop_cbc] =
+    MPSolver::OptimizationProblemType::CBC_MIXED_INTEGER_PROGRAMMING;
+
+            */
+}
+
+bool OrtoolsUtils::isOrtoolsSolverAvailable(OrtoolsSolver ortoolsSolver)
```
Comment (jmkerloch) [inline src/solver/utils/ortools_utils.cpp:451]: This should be the only place where there is changes.

File: src/solver/variable/adequacy/all.h [hunk 8]
```diff
@@ -105,30 +105,30 @@ typedef                             // Prices
   Variable::Adequacy::OverallCost   // Overall Cost (Op. Cost + Unsupplied Eng.)
   <Variable::Economy::OperatingCost // Operating Cost
    <Variable::Economy::Price        // Marginal price
-                             // CO2
+                                    // CO2
```
Comment (jmkerloch) [inline src/solver/variable/adequacy/all.h:108]: file not changed

**Review by jmkerloch — CHANGES_REQUESTED** [review]
There are too many files that weren't changed by this release.
Clang must be passed again for these files.

File: INSTALL.md (hunk 1 above)
Comment (guilpier-code) [reply INSTALL.md]: C'est fait

File: src/libs/antares/jit/jit.hxx (hunk 2 above)
Comment (guilpier-code) [reply src/libs/antares/jit/jit.hxx:71]: It was passed, but my clang-format is probably not exactly the same as the one run on branch develop.

File: src/libs/antares/study/parts/hydro/container.cpp (hunk 3 above)
Comment (guilpier-code) [reply src/libs/antares/study/parts/hydro/container.cpp:177]: Strange indeed, but my answer is the same as previouly.

File: src/libs/antares/study/study.cpp (hunk 4 above)
Comment (guilpier-code) [reply src/libs/antares/study/study.cpp:558]: same answer :-)

File: src/solver/simulation/solver.hxx (hunk 5 above)
Comment (guilpier-code) [reply src/solver/simulation/solver.hxx]: I passed dos2unix on that file, this solves the problem

File: src/solver/utils/ortools_utils.cpp (hunk 7 above)
Comment (guilpier-code) [reply src/solver/utils/ortools_utils.cpp]: I passed dos2unix on that file, this solves the problem

**Pull request merged**